lib/runtime-core/src/import.rs

```diff
@@ -79,6 +80,46 @@ impl ImportObject {
     pub fn get_namespace(&self, namespace: &str) -> Option<&(dyn LikeNamespace + 'static)> {
         self.map.get(namespace).map(|namespace| &**namespace)
     }
+
+    pub fn merged(mut imports_a: ImportObject, mut imports_b: ImportObject) -> Self {
```

Comment:
Can you also add some documentation here about the behavior of this method during edge cases (e.g. Two items with the same name)?